core/serialization/blocks.js

```diff
@@ -171,14 +174,33 @@ const saveExtraState = function(block, state) {
   }
 };
 
+/**
+ * Adds the state of all of the icons on the block to the given state object.
+ * @param {!Block} block The block to serialize the icon state of.
+ * @param {!State} state The state object to append to.
+ */
```

Comment:
I don't think I need to do that in this file because I've built it as a goog.module. Here's an excerpt from #5026
> Any top-level `@private` or otherwise internal identifiers should become top-level, module-local identifiers; they do not get marked `@private` nor do their names end with underscore...

But I'm not sure.

Reply:
Ack no you are correct. Sorry forgot for a second we were in goog.module land.

Reply:
It's so hard haha. Especially with samples being mostly class-based, there are like 3-4 different partial style guides lol.